web/tests/EducationBenchmarking.Web.E2ETests/Pages/CompareYourCostsPage.cs

```diff
@@ -273,9 +288,11 @@ await accordionState
         }
     }
 
-    public async Task AssertDropDownDimensions(ILocator dimensionsDropdown,string[] expectedOptions )
+    public async Task AssertDropDownDimensions(ILocator dimensionsDropdown, string[] expectedOptions)
     {
-        string[] actualOptions = await dimensionsDropdown.EvaluateAsync<string[]>("(select) => Array.from(select.options).map(option => option.value)");
-          Assert.Equal(actualOptions, expectedOptions);
+        string[] actualOptions =
```

Comment:
Consider using var

Reply:
if we know its of string type then what benefit would it add to use var?